flink-yarn-tests/src/test/java/org/apache/flink/yarn/YARNSessionFIFOITCase.java

```diff
@@ -276,19 +280,21 @@ public void testfullAlloc() throws IOException {
 	 */
 	@Test
 	public void testJavaAPI() throws Exception {
```

Comment:
How about we delete `testJavaAPI`? This test only starts a session cluster and tests whether the `ClusterConnectionInfo` and the URL of the web UI are not null. I don't see how this is related to Yarn's FIFO scheduler, i.e., the test does not belong here. Retrieving the `ClusterConnectionInfo` and URL of the web ui is already covered by other tests [1][2]. Lastly, the ported version in this PR does not have the same assertions as before. Let me know what you think, @TisonKun.

[1] https://github.com/apache/flink/blob/0a54983ca0207444477cfe991dbee1b611496204/flink-clients/src/test/java/org/apache/flink/client/cli/DefaultCLITest.java#L71
[2] https://github.com/apache/flink/blob/0a54983ca0207444477cfe991dbee1b611496204/flink-clients/src/main/java/org/apache/flink/client/program/rest/RestClusterClient.java

Reply:
Thanks for your investigation. I've checkout it and agree with you, adding a follow-up to address your comment.

As a following consideration, we should properly test FLINK on YARN in the future. Current status is not quite robust and elegant IMO (ó﹏ò｡)